src/components/SidebarItems.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import BarChartIcon from '@material-ui/icons/BarChart';
+import HomeIcon from '@material-ui/icons/Home';
+
+export default function SidebarItems() {
+  return(
+    <div>
+      <ListItem button onClick={() => window.open("http://localhost:3000/", "_self")}>
```

Comment:
Sugiro colocar essa URL numa const externa para facilitar o deploy

Reply:
Se são links internos, deveria-se usar um router

Reply:
Atualizei criando dois secrets: BASE_URL e ANALYTICS_URL. Pra rodar local só criar como comentei na descrição. O Admin do Repositório ainda precisa criar antes de fazer o deploy.

Ja tinha deixado como outra atividade "to improve" pra alguem (pode ser eu tambem kk) fazer o roteamento (acho melhor  esperar refatorar a página "metrics.html" ainda.